INSTALL.asciidoc

```diff
@@ -45,15 +60,27 @@ cd /tmp/os-autoinst-run
 --------------------------
 7.  create a minimal "vars.json" config file
 +
+.vars.json
+[source,json]
 ---------------------------------------------------------
-cat <<EOF >vars.json
 {
-    "DISTRI":  "sle",
-    "CASEDIR": "/space/SVN/os-autoinst-distri-opensuse/",
-    "BACKEND": "s390x"
+   "ARCH" : "x86_64",   
+   "BACKEND" : "qemu",
+   "CASEDIR" : "/path/to/os-autoinst-distri-opensuse",
+   "CDMODEL" : "virtio-scsi-pci",
```

Comment:
I think most of it can be omitted now that I know a bit more about os-autoinst. Also I think it would be more appropriate to use something like: https://github.com/os-autoinst/os-autoinst-distri-example/ here because this guide should stay focused on the backend to limit overlap with the OpenQA documentation.